src/Wallabag/CoreBundle/Helper/EntriesExport.php

```diff
@@ -162,7 +162,7 @@ private function produceEpub()
 
         $book->setTitle($this->title);
         // Not needed, but included for the example, Language is mandatory, but EPub defaults to "en". Use RFC3066 Language codes, such as "en", "da", "fr" etc.
-        $book->setLanguage($this->language);
+        $book->setLanguage(substr($this->language, 0, 2));
```

Comment:
Even though https://github.com/wallabag/PHPePub/pull/3#discussion_r825345271 will fix an issue on PHPePub side, we still need to fix something in wallabag, here is a suggestion @j0k3r as we store language with `_`:

```suggestion
        // EPub specification requires BCP47-compliant languages, thus we replace _ with -
        $book->setLanguage(str_replace('_', '-', $this->language));
```